.travis.yml

```diff
@@ -431,7 +432,7 @@ deploy:
 
   - provider: script
     edge: true # This supposedly opts in to deploy v2.
-    script: ./ci/keep_alive bash $TRAVIS_BUILD_DIR/ci/travis/build-autoscaler-images.sh || true
+    script: ./ci/keep_alive bash $TRAVIS_BUILD_DIR/ci/travis/build-autoscaler-images.sh
```

Comment:
in a sense, this is really broader than autoscaler now (it builds rayproject/base-deps and rayproject/ray) -- perhaps `git mv ci/travis/build-autoscaler-images.sh ci/travis/build-docker-images.sh` (and fix references)

Reply:
Great idea--this makes a lot of sense now :)